spf: add page and Chinese translation

yorukot opened the pull request:
<!--
Thank you for contributing!
Please fill in the following checklist, removing items that do not apply.
See also https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md
-->

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=17009) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=17009) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=17009) it.</sub>

spageektti reviewed (commented):
Hi. Thanks for your contribution.

spageektti commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+`spf --fix-config-file`
+
+- Use specific config and hotkey files:
```
```suggestion
- Use specific configuraration and hotkey files:
```

spageektti commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+`spf --chooser-file {{/tmp/chooser-result}}`
+
+- Show internal config and data directory paths:
```
```suggestion
- Show internal configuration and data directory paths:
```

spageektti commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+# spf
+
+> superfile – Modern terminal file manager.
```
```suggestion
> Superfile – Modern terminal file manager.
```

yorukot replied:
Thanks! The official name is actually superfile (lowercase), so I kept it that way intentionally.
That said, I know `tldr` descriptions are supposed to start with a capital letter. Do you have any suggestions on how we could phrase it while still keeping the lowercase name visible?

yorukot replied:
Yeah sure great idea!

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+# spf
+
+> The superfile – Modern terminal file manager.
+> More information: <https://github.com/yorukot/superfile>.
```
Does superfile has direct CLI usage documentation? I'm unable to find it with quick lookup.

yorukot replied:
No, it's not mentioned in the documentation.
But I can show you what it looks like when you run the CLI:
```
NAME:
   superfile - A new cli application

USAGE:
   superfile [global options] command [command options] [path]

VERSION:
   v1.3.1

DESCRIPTION:
   Pretty fancy and modern terminal file manager

COMMANDS:
   path-list, pl  Print the path to the configuration and directory
   help, h        Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --fix-hotkeys, --fh               Adds any missing hotkeys to the hotkey config file (default: false)
   --fix-config-file, --fch          Adds any missing hotkeys to the hotkey config file (default: false)
   --print-last-dir, --pld           Print the last dir to stdout on exit (to use for cd) (default: false)
   --config-file value, -c value     Specify the path to a different config file
   --hotkey-file value, --hf value   Specify the path to a different hotkey file
   --chooser-file value, --cf value  On trying to open any file, superfile will write to its path to this file, and exit
   --help, -h                        show help
   --version, -v                     print the version
```

Managor replied:
Thanks.

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+- Fix hotkey settings by appending missing keys:
+
+`spf --fix-hotkeys`
```
In the CLI usage documentation I would have liked to check if there are short options of these.

Managor replied:
```suggestion
`spf {{[-fh|--fix-hotkeys]}}`
```

Managor replied:
Add the help page in the front page readme.md so that it's easily accessible withoyt needing to install the program.

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+- Fix the configuration file by appending missing entries:
+
+`spf --fix-config-file`
```
```suggestion
`spf {{[--fch|--fix-config-file]}}`
```

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+- Use specific configuration and hotkey files:
+
+`spf --config-file {{path/to/config.toml}} --hotkey-file {{path/to/hotkey.toml}}`
```
```suggestion
`spf {{[-c|--config-file]}} {{path/to/config.toml}} {{[--hf|--hotkey-file]}} {{path/to/hotkey.toml}}`
```

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+- Set a chooser file: write the opened path to this file and exit:
+
+`spf --chooser-file {{/tmp/chooser-result}}`
```
```suggestion
`spf {{[--cf|--chooser-file]}} {{/tmp/chooser-result}}`
```

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+- Show internal configuration and data directory paths:
+
+`spf path-list`
```
```suggestion
`spf {{[pl|path-list]}}`
```

Managor reviewed (changes requested):
These will allow the client to show either new user friendly long versions or power user friendly short versions

Managor commented on pages/common/spf.md:
```diff
@@ -0,0 +1,32 @@
+
+- Fix hotkey settings by appending missing keys:
+
+`spf {{[-fh|--fix-hotkeys]}}`
```
```suggestion
`spf {{[--fh|--fix-hotkeys]}}`
```
My mistake.

yorukot commented:
Hi! I’ve updated the PR based on your feedback.
- I’ve adjusted the request as suggested.
- Also, I revised the description for the `--chooser-file` option to make it clearer.

Let me know if there’s anything else that should be improved. Thanks again for your review!

Managor commented:
Next time tmit would be nice to first submit just the english page and then make a second PR for translations once the page is pulled. That way you have to hash out these things only once.

Managor commented:
I'll go ahead and merge. Thank you for your first contribution and welcome to the project. Send a PR any time to adjust these pages if you feel like they could be better.
